meson.build

```diff
@@ -27,6 +27,13 @@ if api_bus == 'user'
    common_cflags += '-DUSE_USER_API_BUS'
 endif
 
+add_project_link_arguments('-lgcov', language : 'c')
```

Comment:
You can move this into `if` since we only need this linker argument when building with coverage.

Reply:
Not *you can*, but *you need to* is the right term :-;